ziti/config.go

```diff
@@ -0,0 +1,67 @@
+/*
+	Copyright 2019 NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package ziti
+
+import (
+	"encoding/json"
+	"github.com/openziti/identity"
+	"github.com/pkg/errors"
+	"os"
+)
+
+type Config struct {
+	//ZtAPI should be in the form of https://<domain>[:<port>]/edge/client/v1
```

Comment:
another opportunity along with "zt-session" for a constant imo...  i don't think we have a good place for constants like this though

Reply:
The edge does have those constants already. 

The edge depends on the SDK, so they would have to be here or in some other repo that isn't the edge. It is also kinda weird to move them out of edge as that is where their primary use is.